Source/VCGeneration/StratifiedVC.cs

```diff
@@ -14,6 +14,9 @@ namespace VC
 {
   using Bpl = Microsoft.Boogie;
 
+  /// <summary>
+  /// Used in Corral https://github.com/boogie-org/corral
+  /// </summary>
```

Comment:
Unless that class only exists for that purpose, should this really be an XML doc comment?

Reply:
Do you know of other use cases?

Reply:
I've only found it to be used in Corral.

Reply:
I was suggesting to convert it to a regular comment.

Reply:
Okay, why? It serves as a (poor) explanation of why this exists, not how it works.